Add `The Followpoint Special: Behind the Banhammer` news post (#13019)
ppy/osu-wiki: news/2025/2025-03-25-the-followpoint-special-behind-the-banhammer.md

---
layout: post
title: "The Followpoint Special: Behind the Banhammer"
date: 2025-03-25 17:00:00 +0000
series: The Followpoint
---

> Ever wanted to get some insight into how Global Moderation Team works? Then now is your chance as we've interviewed the GMT leaders themselves, so come and have a look at our (sometimes silly) questions!

Reviewer: ```suggestion
Ever wanted to get some insight into how [Global Moderation Team](/wiki/People/Global_Moderation_Team) works? Then now is your chance as we've interviewed the GMT leaders themselves, so come and have a look at our (sometimes silly) questions!
```

Author: I don't see a point in linking it in short intro tbh + it was already linked a little below

Reviewer: yeah i thought it would work but at the same time i had to made that change just in case

![](/wiki/shared/news/2025-03-25-the-followpoint-special-behind-the-banhammer/banner.jpg)

> Welcome, strangers and returning readers! Like you've already read in the title and the short intro, this The Followpoint post will be an unusual one, talking about a team inside osu! instead of an individual person. [Global Moderation Team](/wiki/People/Global_Moderation_Team) (usually referred to as GMT) is one of the essential and oldest osu! teams, consisting of community members who mainly focus on player moderation. Its [wiki article](/wiki/People/Global_Moderation_Team) explains things really well, so you should give it a read if you haven't yet!

Reviewer 2: 'this The Followpoint' wording feels pretty odd, i'd probably drop the 'The'

Hello, for starters let's do a question about history! In the past GMT were called Administrators, but why specifically?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3227533" style="background-image: url('/wiki/shared/avatars/Kobold84.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/3227533">Kobold84</a></p>

At one point Admins had the same usergroup (<https://osu.ppy.sh/groups/4>) we have now, but that was from before I even heard of the game. Some curious users could refer to this page (<https://web.archive.org/web/20120904074341/https://osu.ppy.sh/wiki/GMT>) but this group didn't really have anything to do with the GMT. As for the term itself, "Administrators" must have been used only because of the phpBB default user title for users with the admin control panel.

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>

> Why are GMT members' names red in osu! Stable chat despite the usergroup having green color?

Reviewer: ```suggestion
Why are GMT members' names red in osu!(stable) chat despite the usergroup having green colour?
```

Author: Agreed on osu!(stable), but not sure if "color" needs to be changed, since technically we are showing the interview like a chat in Discord for example

Reviewer: color is US English, while UK English is colour, thats why i had to change that since most articles and newspost contains UK English

Reviewer: at the same time it works for both but i would rather use UK one

Author: I'm mentioning this because usually such things aren't changed in the interview, probably best to ask @Walavouchey

Yes. The current rules page was reworked in 2017 and the GMT had a more direct control over what was on it. We gathered suggestions for specific rule changes and had a vote afterwards. Most of the time the implemented changes haven't been major rules changes, the vibes are out of our control, after all we don't set the vision of how the community should be moderated, we only upheld it.

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>

Since GMT [usergroup page](https://osu.ppy.sh/groups/4) refers to NAT as their sibling team, who do you think would come out on top if these two were put into a boxing ring?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/7944724" style="background-image: url('/wiki/shared/avatars/0x84f.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/7944724">0x84f</a></p>

> We’d whoop their butts any day of the week. If we were to just put the leaders one up against another, put me against Hivie and I’d KO him without breaking a sweat.

Reviewer: ```suggestion
We’d whoop their butts any day of the week. If we were to just put the leaders one up against another, put me against [Hivie](https://osu.ppy.sh/users/14102976) and I’d KO him without breaking a sweat.
```

It can be either of those two, or both. I’d say the more common scenario we have is that we have members that speak the language, but the chat just isn’t active. There are some exceedingly rare cases where some channels need moderation without active moderators, but we are actively working on closing those gaps.

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>

Speaking of which, I've heard of GMT applications opening for people who speak a specific language in the past, to get more people for channels that lack moderation, but how and who usually decides that a specific language needs more moderators?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/5999631" style="background-image: url('/wiki/shared/avatars/Venix.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/5999631">Venix</a></p>

> Usually, it's just somebody from the team pointing out the critical lack of moderators in a certain area; you know — it's not that hard to notice when things are getting out of hand somewhere.

Reviewer 3: ```suggestion
Usually, it's just somebody from the team pointing out the critical lack of moderators in a certain area — you know — it's not that hard to notice when things are getting out of hand somewhere.
```

For regular day-to-day additions it usually works the opposite way: if we have a person be active in a channel or a certain area, their candidature will be likely discussed when they are here rather than specifying a need for moderation of a certain field and then considering the addition.

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>

Do you have any plans to have a permanent application system in the future, like what BNs have on their website?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3227533" style="background-image: url('/wiki/shared/avatars/Kobold84.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/3227533">Kobold84</a></p>

The short answer is we don't have "plans" to have them permanent.

> The longer answer is that we've had this suggestion from time to time, and it seems enticing, since the workload is huge if we go through all applicants at once. However, this suggestion hasn't had a lot of support, and one of the main reasons I can think of is that it's important for our team to have a huge influx of "fresh blood" to lessen the disconnect between the community and the GMT, this way there's a natural rotation and no stagnant air. From my experience with all of the apps it does seem true, with each new round iteration you could see positive changes within the team after bringing in dozens of new members all at once. Hopefully, the upcoming apps will work out the same way.

Reviewer 3: ```suggestion
The longer answer is that we've had this suggestion from time to time, and it seems enticing, since the workload is huge if we go through all applicants at once. However, this suggestion hasn't had a lot of support, and one of the main reasons I can think of is that it's important for our team to have a huge influx of "fresh blood" to lessen the disconnect between the community and the GMT. This way there's a natural rotation and no stagnant air.

From my experience with all of the apps it does seem true that with each new round, you could see positive changes within the team after bringing in dozens of new members all at once. Hopefully, the [current apps](https://osu.ppy.sh/home/news/2025-03-09-gmt-apps-now-open) will work out the same way.
```

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>

Are there any specific misconceptions about the GMT's responsibilities or authority that you think players often misunderstand and you'd like to clarify?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/5999631" style="background-image: url('/wiki/shared/avatars/Venix.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/5999631">Venix</a></p>

> Once in a while, you can come across the statements that the Global Moderation Team is responsible for handling account restrictions, silence appeals, or account-related inquiries in general. As a matter of fact though, this belongs to the [account support team](/wiki/People/Account_support_team) set of tasks.

Reviewer 3: ```suggestion
Once in a while, you can come across the statements that the Global Moderation Team is responsible for handling account restrictions, silence appeals, or account-related inquiries in general. As a matter of fact though, this belongs to the [account support team](/wiki/People/Account_support_team)'s set of tasks.
```

> If you are facing any issues with your account, please contact the article above and direct your concerns to one of the outlined inboxes.

Reviewer 4: ```suggestion
If you are facing any issues with your account, please consult the article above and direct your concerns to one of the outlined inboxes.
```

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>

And lastly, what is your favorite GMT out of context quote?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/7944724" style="background-image: url('/wiki/shared/avatars/0x84f.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/7944724">0x84f</a></p>

"Hey, the."

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3227533" style="background-image: url('/wiki/shared/avatars/Kobold84.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/3227533">Kobold84</a></p>

"Hey, the."

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/5999631" style="background-image: url('/wiki/shared/avatars/Venix.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/5999631">Venix</a></p>

"Hey, the."

</div>

---

> I'd like to thank all Global Moderation Team members for everything they do for osu! and their efforts to make chats a nicer place! I know that sometimes it doesn't seem like it, but they are really trying their best, so please don't let some bad apples from the past make you think all GMTs are like this, I know some of them myself and they are amazing people. <3

Reviewer 3: is this referring to the people you just interviewed ("like this") or some random event months/years ago? either way this sounds passive-aggressive and confusing

```suggestion
I'd like to thank all Global Moderation Team members for everything they do for osu! and their efforts to make chats a nicer place! I know that sometimes it doesn't seem like it, but they are really trying their best. I know some of them myself and they are amazing people. <3
```

> And of course thank you 0x84f, Kobold84 and Venix for agreeing to this interview and answering my questions, I'm really glad I was given this opportunity to show a wider audience a bit more of what GMT really is!

Reviewer 2: 'of what **the** GMT really is'

> And of course thank you 0x84f, Kobold84 and Venix for agreeing to this interview and answering my questions, I'm really glad I was given this opportunity to show a wider audience a bit more of what the GMT really is!

Reviewer 3: ```suggestion

And of course thank you 0x84f, Kobold84 and Venix for agreeing to this interview and answering my questions, I'm really glad I was given this opportunity to show a wider audience a bit more of what the GMT really is!
```

> Now I'll say a few words about The Followpoint itself. After our [last interview](https://osu.ppy.sh/home/news/2024-12-05-the-followpoint-hivie-the-man-of-many-things) things slowed down due to a lot of things happening in our lifes, but now as we recover — things should get back to normal. Anyway, I really hope you enjoyed this special interview, and thank you for reading it!

Reviewer 3: ```suggestion
Now I'll say a few words about The Followpoint itself. After our [last interview](https://osu.ppy.sh/home/news/2024-12-05-the-followpoint-hivie-the-man-of-many-things) things slowed down due to a lot of things happening in our lives, but now as we recover, things should get back to normal. Anyway, I really hope you enjoyed this special interview, and thank you for reading it!
```

> Right now we cannot promise the next interview will come out on schedule like before, but please look forward to a new interview coming out hopefully soon enough, and make sure to keep an eye on the news feed! You may also follow us [on Bluesky](https://bsky.app/profile/thefollowpoint.megamix.dev) or [on Twitter](https://twitter.com/The_Followpoint) to see more updates and question submissions for future interviews.

Reviewer 3: may want to consider not promoting twitter/bsky on single every post in future iterations (we don't do this kind of thing elsewhere)

Author: Fair, in this newspost I think it makes sense considering we had a big pause, but in the future yeah, I agree.

At one point Admins had [the same usergroup](https://osu.ppy.sh/groups/4) we have now, but that was from before I even heard of the game. Some curious users could refer to [this page](https://web.archive.org/web/20120904074341/https://osu.ppy.sh/wiki/GMT) but this group didn't really have anything to do with the GMT. As for the term itself, "Administrators" must have been used only because of the phpBB default user title for users with the admin control panel.

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>

Why are GMT members' names red in osu!(stable) chat despite the usergroup having green colour?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3227533" style="background-image: url('/wiki/shared/avatars/Kobold84.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/3227533">Kobold84</a></p>

> IRC shenanigans. Due to the chat system the stable client is integrated with, all users with an operator status (mod powers) are red, so it's not just us who are red! Admins and devs also have this color and of course the full-red NAT.

Reviewer 5: are we going for factual accuracy here or just going with what people think they know about the game's history? because this doesn't sound right to me.

Author: Well, that's what Kobold thinks is the reason, but feel free to correct us if that's not why

Author: Not sure how to go about this in the text itself tho, will have to ask Kobold I guess

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>

Speaking of which, how many "Slowpokes" are there currently?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/5999631" style="background-image: url('/wiki/shared/avatars/Venix.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/5999631">Venix</a></p>

1386 and counting.

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>

> While many players are aware of the weekly osu!dev catch-up meetings on the official server, little is known about the internal discussions or processes within the GMT itself. So, can you tell us how your team meetings typically work?

Reviewer 5: bi weekly if we wanna be accurate
